packages/next-frontend/src/components/step/preview/index.tsx

```diff
@@ -7,7 +7,7 @@ import { Typography } from "@/src/ui/typography";
 import styles from "./styles.module.css";
 
 export interface StepPreviewProps {
-    label: string;
+    label: string | ReactNode;
```

Comment:
The `ReactNode` type already includes `string` in its definition

Reply:
Fixed in 73fee9a5f735c6ec90dd504022666e8b724a0de9.